torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3087,38 +3087,79 @@ def generator():
 
     return list(generator())
 
+class _TestParamsMaxPool1d(object):
+
+    def __init__(self):
+        self.kwargs = {
+            'kernel_size': [(2,), 3],
+            'stride': [2],
```

Comment:
Do you think testing `stride=None` will be useful as well?

Reply:
Makes sense!